testdata/terraform/cloudflare_ruleset_zone_http_request_firewall_custom/test.tf

```diff
@@ -15,6 +15,7 @@ resource "cloudflare_ruleset" "terraform_managed_resource" {
     }
     logging {
       enabled = true
```

Comment:
`enabled = (bool)` shouldn't be present as they were the old approach before `status = "..."`; you'll want to unset them in the generator.